Makefile

```diff
@@ -69,10 +67,10 @@ unit-test: $(SOURCES)
 	go test -v -cover -race -timeout 10s ./pkg/... ./plugins/...
 
 integration-test: $(SOURCE)
-	go test -v -tags integration -race -timeout 10s ./pkg/... ./plugins/...
+	go test -v -tags integration -race -gcflags=all=-d=checkptr=0 -timeout 10s ./pkg/... ./plugins/...
 
 sudo-integration-test: $(SOURCE)
-	sudo -E ${GO_EXECUTABLE} test -v -tags "sudo integration" -race -timeout 10s ./plugins/...
+	sudo -E ${GO_EXECUTABLE} test -v -tags "sudo integration" -race -gcflags=all=-d=checkptr=0 -timeout 10s ./plugins/...
```

Comment:
Do these errors exist currently as well? Are checkptr errors related to same boltdb issue as in this PR - https://github.com/aws/amazon-ecs-cni-plugins/pull/106? Can a boltdb version change fix it, as in that PR?

Reply:
Ooh, thank you for the reference. The `checkptr` errors from BoltDB are the same as the ones in https://github.com/aws/amazon-ecs-cni-plugins/pull/106, and they still exist on `master` because that PR was never merged.

https://github.com/aws/amazon-ecs-cni-plugins/pull/113/commits/7ac5bce70c595c782689f28c15a1a0b00601204b upgrades `docker/libkv` to the latest commit (it's the same as what's in [#106](https://github.com/aws/amazon-ecs-cni-plugins/pull/106/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R10)), and that still resolves the BoltDB issues today.